### PR TITLE
Specify how provide a high availability

### PR DESCRIPTION
**Pull request by jderusse**
Fixes https://github.com/symfony/symfony/pull/26026

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
I like this, but it's a bit confusing when reading it. Could we reword it to something like this?

```
    In order to get high availability when using the ``ConsensusStrategy``, the
    minimum cluster size must be three servers. This allows the cluster to keep
    working when a single server fails (because ``ConsensusStrategy`` requires
    that the response is the same in more than half of the servers).
```

**Comment by jderusse** [comment]
I totally agree with your suggestion. (my English is far to be perfect :-), try to do my best)

I'm on my phone, do you mind to update my PR (or push a new one)? If not I would update it tonight with your proposal
